### ISSUE TITLE
ERROR: --dest must be a bare name

### ISSUE THREAD

**Issue by ngtuna** [kubernetes/git-sync#72]
I ran the example on README and it gave me this below error:

```
$ docker run -d -v /tmp/git-data:/git gcr.io/google-containers/git-sync-amd64:v2.0.5-13-g267a78b --repo=https://github.com/ngtuna/k8s_notes --branch=master --wait=30
827186f9ea9ad2194eb252386126722c980bb707460a951b179122ab7b53b833

Tunas-MacBook-Pro:git-sync tuna$ docker logs -f 827186f9ea9ad2194eb252386126722c980bb707460a951b179122ab7b53b833
ERROR: --dest must be a bare name
Usage of /git-sync:
```

**Comment by ngtuna** [comment]
Ok I can pass it by set the `--dest` flag to a leaf dir of /git. I am not sure if this is a bug, seem like we can't sync the git repo into the default /git folder.

**Comment by thockin** [comment]
In order to do atomic updates, it has to be in a subdir.  There's a
proposal to offer non-atomic behavior, but it isn't implemented yet.

**Comment by alexryan** [comment]
How exactly should the --dest flag to a leaf dir of /git be passed in order to work-around this error?

**Comment by thockin** [comment]
`--dest=dirname` will make the final git content be in a subdir named
"dirname".

**Comment by frankfarzan** [comment]
This should be fixed by PR #82

**Comment by stp-ip** [comment]
Fixed in latest release.
